pkg/utils/utils.go

```diff
@@ -165,6 +166,7 @@ func ComputeTrieValue(l4Info []v1alpha1.Port, log logr.Logger, allowAll, denyAll
 
 	for _, l4Entry := range l4Info {
 		if startOffset >= TRIE_VALUE_LENGTH {
+			log.Error(errors.New("trie value exceeds max trie value length"), "trie value exceeds the 24 port/protocol combinations")
```

Comment:
`trie` might not be intuitive for the log msg. Instead we can use - `No.of unique port/protocol combinations supported for a single endpoint exceeded the supported maximum of 24`.